pkg/agent/openflow/client.go

```diff
@@ -1539,3 +1535,18 @@ func (c *client) UninstallMulticlusterPodFlows(podIP string) error {
 	}
 	return c.deleteFlows(c.featureMulticluster.cachedPodFlows, podIP)
 }
+
+func GetFlowModMessages(flows []binding.Flow, op binding.OFOperation) []*openflow15.FlowMod {
+	messages := make([]*openflow15.FlowMod, 0, len(flows))
+	for i := range flows {
+		bundleMessages, _ := flows[i].GetBundleMessages(op)
+		msg := bundleMessages[0].GetMessage().(*openflow15.FlowMod)
+		messages = append(messages, msg)
+	}
+	return messages
+}
+
+func getFlowModMessage(flow binding.Flow, entryOper binding.OFOperation) *openflow15.FlowMod {
```

Comment:
```suggestion
func getFlowModMessage(flow binding.Flow, op binding.OFOperation) *openflow15.FlowMod {
```
for consistent naming